Gruntfile.js

```diff
@@ -133,7 +133,7 @@ module.exports = function(grunt) {
     babel: { 
       options: { 
         sourceMap: true, 
-        presets: ['es2015']
+        presets: ['@babel/preset-env']
```

Comment:
Are these `@babel` references referring to to something within Babel itself, or some config file I don't see?

Reply:
It refers to the namespaced plugin that Babel switched to